plenum/common/messages/node_messages.py

```diff
@@ -271,6 +272,20 @@ class ViewChangeDone(MessageBase):
     )
 
 
+class CurrentState(MessageBase):
+    """
+    Node sends this kind of message for nodes which 
+    suddenly reconnected (lagged). It contains information about current 
+    pool state, like view no, primary etc.
+    """
+    typename = CURRENT_STATE
+
+    schema = (
+        (f.VIEW_NO.nm, NonNegativeNumberField()),
+        (f.PRIMARY.nm, IterableField(AnyField())),  # ViewChangeDone
```

Comment:
Added TODO about reimplementing validation so that it can work with internal messages